lib/utils.js

```diff
@@ -167,8 +167,9 @@ exports.findGitRoot = function (start) {
 exports.findProjectRoot = function (start) {
 
     start = start || Path.dirname(internals.findParent(module).filename);
-    var position = start.indexOf('node_modules');
-    var root = start.slice(0, position === -1 ? undefined : position - Path.sep.length);
+    var position = start.indexOf('/node_modules/');
```

Comment:
This was a bug fix to ensure it found the correct node_modules directory, I had a problem where I keep my node modules I am working on in a directory called _node_modules and this was picking up the incorrect directory